js/settings.json

```diff
@@ -5,10 +5,15 @@
   ],
   "tools": [
     "pencil",
+<<<<<<< HEAD
+    "markerBrush"
+=======
     "oscCircles"
+>>>>>>> 9f3ac3d0f9459ba142d45a8d430e3e55e1251c4b
```

Comment:
lines 8, 10 && 12 were probably left over when u last pulled changes from upstream. if/when git notices a "merge conflict" it will add lines like this to ur code to let u know that u need to resolve the conflict. in this case all u need to do is remove these lines (&7 add a `,` after ur "markerBrush"